### ISSUE TITLE
Add support for DecSync

### ISSUE THREAD

**Issue by shbach** [readrops/Readrops#73]
[DecSync](https://github.com/39aldo39/DecSync) is quite a different service and supporting it would add greater diversity to services supported by Readrops. Furthermore, only [one other android app](https://github.com/39aldo39/spaRSS-DecSync) currently supports DecSync and it's honestly not that great so adding this would make Readrops stand out among the current feed readers on android.

**Comment by Shinokuni** [comment]
Indeed, DeSync is quite different from current services supported by Readrops. I think Readrops will first support other similar services, and then, add support for DeSync.

**Comment by machiav3lli** [comment]
I wouldn't count it as a service but rather as a backup/auto-sync solution for the feeds.

The mentioned app is just a forked spaRSS with the added support for DecSync. This is more than a great setup to re-implement a feature.

**Comment by machiav3lli** [comment]
And yeah, this would solve #72
